.readthedocs.yml

```diff
@@ -0,0 +1,19 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-22.04
```

Comment:
```suggestion
  os: "ubuntu-22.04"
```

Reply:
I don't think that is the problem. The string `ubuntu-22.04` is also listed w/o quotes here:

https://docs.readthedocs.io/en/stable/config-file/v2.html

Reply:
Also, https://www.yamllint.com/ claims the `.readthedocs.yaml` files is indeed valid YAML.

Reply:
Just noting that it DOES work for https://github.com/conda/conda-build/pull/5006/files#diff-cde814ef2f549dc093f5b8fc533b7e8f47e7b32a8081e0760e57d5c25a1139d9R4

Reply:
(could also just be an unrelated state issue since you're adding the file completely new)